src/super_gradients/training/transforms/detection/detection_longest_max_size.py

```diff
@@ -0,0 +1,63 @@
+import random
+from typing import List
+
+import cv2
+import numpy as np
+
+from super_gradients.common.object_names import Transforms, Processings
+from super_gradients.common.registry import register_transform
+from super_gradients.training.samples import DetectionSample
+from .abstract_detection_transform import AbstractDetectionTransform
+
+
+@register_transform(Transforms.DetectionLongestMaxSize)
+class DetectionLongestMaxSize(AbstractDetectionTransform):
```

Comment:
Why are some transforms under here while others left in transforms.py?
And why are they inheriting differently from the ones there?

Will it work with our current way of passing samples in detectiondataset ?

Reply:
> Why are some transforms under here while others left in transforms.py?
Existing transforms left in original place to make sure PR diff shows meaningful results. We will move those to individual files eventually, but that will happen later.
However new transforms that I introduced in this PR (why I did that is answered next) are placed straight to a new/correct place.

> And why are they inheriting differently from the ones there?
We introduce new `AbstractDetectionTransform` class with new signature (`apply_to_sample` insteada of `__call__`). To keep backward compatibility, existing transforms also inherit from `LegacyDetectionTransformMixin` which adds support of `__call__` to be compatible with current dataset classes.
However new transformations are not inheriting from LegacyDetectionTransformMixin since we will drop old API anyway so there is no point in inheriting from `LegacyDetectionTransformMixin` for new classes. 
So no, new classes will not work existing dataset.